games/b.yaml

```
@@ -226,6 +226,27 @@
   type: clone
   updated: 2015-05-19
 
+- name: 'Bedlam 2: Absolute Bedlam'
+  development: active
+  images:
+    - https://camo.githubusercontent.com/2f4bb518d09c99a72494c55ec22d56aab0025051d756d749a795c5faef6b5cd2/68747470733a2f2f7777772e6f6c642d67616d65732e72752f666f72756d2f6174746163686d656e74732f6265646c616d322d706e672e3234313733322f
+  video:
+    youtube: _pjFmPXNnFw
+  lang:
+    - Assembly
+    - C++
+  framework:
+    - SDL2
+  license:
+    - As-is
+  content: commercial
+  originals:
+    - Bedlam 2: Absolute Bedlam
```

```suggestion
    - 'Bedlam 2: Absolute Bedlam'
```
also need to add the original in originals/b.yaml